src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -54,6 +50,17 @@ LRESULT CALLBACK HwndTerminal::HwndTerminalWndProc(
     LPARAM lParam) noexcept
 try
 {
+    if (WM_NCCREATE == uMsg)
+    {
+#pragma warning(suppress : 26490) // Win32 APIs can only store void*, have to use reinterpret_cast
+        auto cs = reinterpret_cast<CREATESTRUCT*>(lParam);

```

Comment:
BTW you can now use `std::bit_cast` for this. But it's not particularly important.